conda_forge_tick/utils.py

```diff
@@ -182,6 +180,11 @@ def __enter__(self) -> "LazyJson":
     def __exit__(self, *args: Any) -> Any:
         self._dump()
 
+    def __update__(self, data: dict) -> None:
```

Comment:
I don't think this needs to be `__update__` it should be `update`

Reply:
I will change that thanks :+1: